packages/common/src/search/types/IHubSearchOptions.ts

```diff
@@ -73,7 +73,7 @@ export interface IHubSearchOptions {
   /**
    * Sort direction
    */
-  sortOrder?: "desc" | "asc";
+  sortOrder?: "desc" | "asc" | "DESC" | "ASC";
```

Comment:
If the  discussion API needs it in caps, transform it internally vs widening the type... b/c that may introduce issues elsewhere.